src/kernels/execution/cellExecution.ts

```diff
@@ -210,7 +210,6 @@ export class CellExecution implements IDisposable {
     public dispose() {
         traceCellMessage(this.cell, 'Execution disposed');
         disposeAllDisposables(this.disposables);
-        this.cellExecutionHandler?.dispose();
```

Comment:
Does someone need to dispose this then? If it's disposable I'd assume that someone would need to call it? Or is someone else already cleaning it up?

Reply:
It is tracked & gets disposed where this is created in `cellExecutionMessageHandlerService.ts`

Reply:
I.e. we don't need to dispose this here.